concepticondata/conceptlists/Heggarty-2017-200.tsv

```diff
@@ -0,0 +1,201 @@
+ID	NUMBER	URL	ENGLISH	DESCRIPTION	CONCEPTICON_ID	CONCEPTICON_GLOSS	EXAMPLE_CONTEXT
+Heggarty-2017-200-172	172	http://cobl.clld.org/parameters/172	thin	'Thin' of an object, not a person.  Opposite of 'thick', not of 'fat'.	2249	THIN (OF SHAPE OF OBJECT)	This piece of wood is thin, and that one is thick.
```

Comment:
For most other concept lists, the ID column is composed of the prefix and consecutive integers rather than the local numbers. Maybe we should use this practice here as well, in particular because the local numbers are not under our control.

Reply:
Okay, will refine.